cmd/channel/email/main.go

```diff
@@ -99,17 +166,39 @@ func (ch *Email) GetInfo() *plugin.Info {
 				"en_US": "SMTP Port",
 				"de_DE": "SMTP Port",
 			},
-			Min: types.Int{NullInt64: sql.NullInt64{Int64: 0, Valid: true}},
+			Min: types.Int{NullInt64: sql.NullInt64{Int64: 1, Valid: true}},
 			Max: types.Int{NullInt64: sql.NullInt64{Int64: 65535, Valid: true}},
 		},
 		{
-			Name: "from",
+			Name: "user",
 			Type: "string",
 			Label: map[string]string{
-				"en_US": "From",
-				"de_DE": "Von",
+				"en_US": "SMTP User",
+				"de_DE": "SMTP Benutzer",
+			},
+			Default: "user@example.com",
+		},
+		{
+			Name: "password",
+			Type: "secret",
+			Label: map[string]string{
+				"en_US": "SMTP Password",
+				"de_DE": "SMTP Passwort",
+			},
+		},
+		{
+			Name:     "encryption",
+			Type:     "option",
+			Required: true,
+			Label: map[string]string{
+				"en_US": "Transport Encryption",
+				"de_DE": "Transportverschlüsselung",
```

Comment:
```suggestion
				"en_US": "SMTP Transport Encryption",
				"de_DE": "SMTP Transportverschlüsselung",
```

All other SMTP related fields are prefixed with "SMTP". I guess, for a user it eliminates some uncertainties.